style.css

```diff
@@ -6,7 +6,16 @@
   box-sizing: border-box;
   font-family: sans-serif, 'Poppins';
   list-style: none;
-  margin-top: 10px;
+  margin-top: 5px;
```

Comment:
- [ OPTIONAL] I would suggest you remove this `margin-top: 5px;`, because it adds a space at the top of your page.